financial-services-accelerator/components/org.wso2.financial.services.accelerator.gateway/src/main/java/org/wso2/financial/services/accelerator/gateway/executor/model/FSAPIRequestContext.java

```diff
@@ -48,7 +48,7 @@ public class FSAPIRequestContext extends RequestContextDTO {
 
     private static final Log log = LogFactory.getLog(FSAPIRequestContext.class);
     private final RequestContextDTO requestContextDTO;
-    private Map<String, Object> contextProps;
+    private Map<String, String> contextProps;
```

Comment:
why did we change this to a string? What if in the future we have a requirement to pass objects around the executor framework? Like for an example we extract the certificate in one executor and set it as a property and do some validations from another executor

Reply:
Yes I agree, Will keep it as <String, Object> map